examples/plotting/zoom_map_example.py

```diff
@@ -0,0 +1,62 @@
+"""
+=========================
+Zoomed-in Cutout of a Map
+=========================
+
+This example demonstrates how to plot a cutout region of a `~sunpy.map.Map` with
+connector lines that indicate the region of interest in the full-disk image.
+"""
+
+import matplotlib.pyplot as plt
+from matplotlib.patches import ConnectionPatch
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+###############################################################################
+# We will start off by defining a region of interest that we want to focus on.
+# In this case, we want to focus on the center of the Sun.
+length = 400 * u.arcsec
+x0 = -300 * u.arcsec
+y0 = -100 * u.arcsec
```

Comment:
On the sample image, maybe the bottom right part (CME eruption?) would be more interesting?

Reply:
Maybe this is near to CME eruption i guess : ) 

![z](https://user-images.githubusercontent.com/72349558/109133871-7276c080-777b-11eb-9008-95d50e4b1e01.png)

Reply:
I think it needs to go more to the right and down.

Reply:
![Screenshot from 2021-02-25 15-20-14](https://user-images.githubusercontent.com/72349558/109135385-072dee00-777d-11eb-9f66-f50b0a0c66da.png)

Reply:
That looks better. I will wait for others to see if they like this before we change this.

Reply:
Okay : )

Reply:
> This looks good to me! +1 on the consistency with the other sample data examples. You could even extend the box a bit to capture the whole eruption

Hey Will thank you for the review. I have made the changes, Please have a look! Also, do we need to change the color of the box and connecting line as box isn't very much visible tho black look elegant still just a thought!
![ss](https://user-images.githubusercontent.com/72349558/110755924-863c1f80-826f-11eb-81b1-fd50e0e2f2fc.png)

Reply:
I wonder if making the box on the left figure white is a good idea.

Also I think the axis labels/titles of the left image should be removed. 
For the right plot, that title and the overlap on text at the top of that image should be resolved.

Reply:
Hey Nabil, is it a good idea to remove latitude, longitude notation as well as AIA_171 image label from left image. It's just my heart isn't accept these changes but I will do.
This one will complete remove the labels as well as ticks `plt.axis('off')`
so i am thinking of  removing them all at once and try putting ticks again using `ax.set_xticks([])`
Other option i tried was these functions but they requires one more argument other than None

> plt.xlabel(' ', None) 
> plt.ylabel(' ', None)
> ax1.set_title(None)

I tried couple of other stuffs from here as well https://docs.astropy.org/en/stable/visualization/wcsaxes/ticks_labels_grid.html 
should we remove all the labels? talking about right one if we are keeping both ticks and label than will have to resize them all. will do it in sometime.

Reply:
I think it makes sense, you don't care about the axis labels or values on the left plot. The focus is the right side plot.

Reply:
Hey Nabil,  how about this : )

![image](https://user-images.githubusercontent.com/72349558/110895874-792b3900-8320-11eb-898b-b48417d196a5.png)

Reply:
The white box stands up better but then they clash with the black lines. I am not sure what to think.

Reply:
I agree! I thought of increasing the size of the box a bit...but now I am thinking of leaving it black. Well white isn't that bad if we ignore certain stuffs that always gonna be there!
What about other things ? labels and all?

Reply:
Yeah just change it back to black.